composer/README.md

```diff
@@ -0,0 +1,74 @@
+# Set up composer
+
+Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you.
+You will need composer for any of Symfony based project.
+
+## 1 Install
+
+Installation procedure should be the same for MacOS and Linux. You can find detailed instructions on https://getcomposer.org/download/ and https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos
+
+By default, the above instructions will install the **latest** composer version.
+As composer2 is not (yet) used on most projects, we will first install composer1 and make it available globally:
```

Comment:
I would just add a note here that the installation code block should be copied from the link above, as this is fail if the hash changes.

Reply:
Version is also hardcoded in install instructions below `--version=1.10.21`. How about just directing the user to the install instructions on getcomposer.org without copy/pasting the commands here, as they can be obsolete pretty quickly?

Reply:
Because those will install composer 2, and we want to install version 1 here?

Reply:
I am aware, but you will pretty much soon get an obsolete version here since its hardcoded. Can we specify e.g. --version=1 ? Or install v2 and downgrade it to get v1?

Reply:
`--version=1` does not work.

We could downgrade before moving it to `~/.local/bin`, I will adjust the instructions.

Reply:
Ok, upgraded the instructions with a new process, where we downgrade in place, copy to `/usr/local/bin`, upgrade back, and then move the version a5194d5

Reply:
Simplified the above a bit in  85e6892 :)

Reply:
Add this after setting the aliases:

If you get an error when executing an alias, add your alias file to `~/.bashrc` or `~/.zshrc`:
```
source ~/.bash_aliases
```